src/pages/Profile.vue

```diff
@@ -1,128 +1,124 @@
+                    ></q-input>
+                  </div>
+                  <div class="col">
+                    <q-input
+                      dense
+                      borderless
+                      v-model="lastname"
+                      :label="$t('lastName')"
+                      class="input"
+                    ></q-input>
+                  </div>
+                </div>
+
+                <div class="row">
+                  <div class="col">
+                    <q-input
+                      dense
+                      borderless
+                      v-model="street"
+                      :label="$t('address.street')"
+                      class="input"
+                    ></q-input>
+                  </div>
+                  <div class="col-3">
+                    <q-input
+                      dense
+                      borderless
+                      v-model="streetNumber"
+                      :label="$t('address.number')"
+                      class="input"
+                    ></q-input>
+                  </div>
+                </div>
+
+                <div class="row">
+                  <div class="col-3">
+                    <q-input
+                      dense
+                      borderless
+                      v-model="zip"
+                      :label="$t('address.zipcode')"
+                      class="input"
+                    ></q-input>
+                  </div>
+                  <div class="col">
+                    <q-input
+                      dense
+                      borderless
+                      v-model="city"
+                      :label="$t('address.city')"
+                      class="input"
+                    ></q-input>
+                  </div>
+                </div>
+
+                <q-btn
+                  class="submit-button"
+                  :label="$t('save')"
+                  :loading="loading"
+                  @click="update">
+                </q-btn>
+              </q-tab-panel>
+
+              <q-tab-panel :name="Tabs.Entries">
+                <my-requests />
+              </q-tab-panel>
+
+              <q-tab-panel :name="Tabs.Offers">
+                <h2>Meine Angebote</h2>
+                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis praesentium cumque magnam odio iure quidem, quod illum numquam possimus obcaecati commodi minima assumenda consectetur culpa fuga nulla ullam. In, libero.</p>
+                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis praesentium cumque magnam odio iure quidem, quod illum numquam possimus obcaecati commodi minima assumenda consectetur culpa fuga nulla ullam. In, libero.</p>
+                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis praesentium cumque magnam odio iure quidem, quod illum numquam possimus obcaecati commodi minima assumenda consectetur culpa fuga nulla ullam. In, libero.</p>
```

Comment:
Is this WIP? If not, please add a `TODO` comment and ideally, also link a JIRA ticket.

Reply:
Will do